pykickstart/commands/ostreesetup.py

```diff
@@ -93,3 +97,61 @@ class RHEL8_OSTreeSetup(F21_OSTreeSetup):
 
 class RHEL9_OSTreeSetup(F21_OSTreeSetup):
     pass
+
+class F38_OSTreeSetup(F21_OSTreeSetup):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.bootc = kwargs.get('bootc', False)
+
+    def __str__(self):
+        retval = super().__str__()
+
+        if self.bootc:
+            retval = retval.strip()
+            retval += " --bootc\n"
+
+        return retval
+
+    def _getParser(self):
+        op = super()._getParser()
```

Comment:
I think we'll have to replace the original _getParser() completely, since this is using the same arguments, but making changing required to false. I think it's going to raise an error if you try to set them again.